### DISCUSSION TITLE
Custom extensions for TipTap editor

### DISCUSSION THREAD

**Discussion by ddresch** [avitorio/outstatic#84] [Ideas]
How is it possible to extend the editor used by outstatic with custom React components?

Example Docs: https://tiptap.dev/examples/interactivity

Is there a way to achieve something like this already?

**Comment by avitorio** [comment]
Hey,
Not yet, but this is definitely on the roadmap. This comes next after we merge the metadata db and custom fields. 

Hopefully we'll be able to support custom react components and plugins soon. I'll create a request for comments about this. Let me know if you have any suggestions.
